change/@ni-nimble-components-6c0ea993-99db-424a-beaa-3efe23616755.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need to worry about any impact on minimum column width? What would happen if you shrunk a column so narrow that there wasn't enough room for the sort icon and then you tried to set it to be sorted?

I'm a bit out of the loop on column sizing and header interaction discussions so it might be that this is only a problem once we support more sizing behaviors. And your change probably doesn't make this any better or worse, just made me think about it.

Reply:
This doesn't have an impact on minimum column width. The idea is that the minimum column width ensure that there is always room for the sort icon to be added, but when the icon is added, the column header becomes truncated more than it previously was.

We also need to update our minimum column width based on the visual design spec.

These designs are captured here: https://xd.adobe.com/view/1a9870c7-2510-4248-83a5-b0148e7a6763-4fcb/